www/kanban/app-kanban.less

```diff
@@ -117,39 +117,39 @@
         }
 
         .kanban-header-yellow {
-            background: #FC3;
+            background: #FC3 !important;
```

Comment:
why is `!important` necessary here ? We do our best to avoid any usage of this.

Reply:
This is because jscolor would otherwise overwrite the default colors with white. It is simply to make sure that default colors have a higher initial priority than jscolor has as no color is yet picked.